src/plugins/notebook/components/notebook.vue

```diff
@@ -367,6 +367,15 @@ export default {
             const notebookStorage = getDefaultNotebook();
             const id = addNotebookEntry(this.openmct, this.internalDomainObject, notebookStorage, embed);
 
+            this.$nextTick(() => {
+                const element = this.$el.querySelector(`#${id}`);
+                if (!element) {
```

Comment:
This shouldn't happen, so just let it error out. At least that way we know something very unexpected happened.

Reply:
updated.